src/capwap_http_api_handler.erl

```diff
@@ -140,34 +158,40 @@ handle_request_wtp(<<"POST">>, [_, <<"set-ssid">> | _], Req, State) ->
             {jsx:encode([{error, badarg}]), Req, State}
     end;
 handle_request_wtp(<<"DELETE">>, [_, <<"stop-radio">> | _], Req, State) ->
-    CN = cowboy_req:binding(id, Req),
+    CN = cowboy_req:binding(wtp_id, Req),
     try
         RadioID = erlang:binary_to_integer(cowboy_req:binding(rid, Req)),
-        Res = capwap_ac:stop_radio(CN, RadioID),
-        {jsx:encode([{res, Res}]), Req, State}
+        case capwap_ac:stop_radio(CN, RadioID) of
+            ok ->
+                {true, Req, State};
+            _ ->
+                {false, Req, State}
+        end
     catch
         _:badarg ->
-            {jsx:encode([{error, badarg}]), Req, State}
+            {false, Req, State}
     end;
 handle_request_wtp(_, _, Req, State) ->
     {jsx:encode([{error, bad_command}]), Req, State}.
 
 handle_request_station(<<"GET">>, [], Req, State) ->
     Stations = capwap:list_stations(),
     Ret = lists:map(fun({{CommonName, Endpoint}, MACs}) ->
-        [{id, CommonName},
-         {address, fmt_endpoint(Endpoint)},
-         {macs, [ fmt_station_mac(Station) || Station <- MACs ]}]
-    end, Stations),
+                        [{id, CommonName},
+                         {address, fmt_endpoint(Endpoint)},
+                         {macs, [ fmt_station_mac(Station) || Station <- MACs ]}]
+                    end, Stations),
     {jsx:encode(Ret), Req, State};
 handle_request_station(<<"DELETE">>, [_], Req, State) ->
-    MACStr = cowboy_req:binding(id, Req),
-    case mac_to_bin(MACStr) of
+    MACStr = cowboy_req:binding(sta_id, Req),
+    case capwap_tools:mac_to_hex(MACStr) of
         MAC when is_binary(MAC) ->
-            R = capwap:detach_station(MAC),
-            {jsx:encode([{result, R}]), Req, State};
```

Comment:
doesn't that change the return type? Doing so might be ok, I'm just not 100% sure about it.